auslib/web/common/releases.py

```diff
@@ -12,6 +12,18 @@
 log = logging.getLogger(__name__)
 
 
+def strip_data(release):
+    """Return a release with all the fields except data.
+
+    This is used in multiple APIs to save bandwidth and present a
+    simplified view of the release by removing its largest field,
+    data, which is of no use except when serving clients.
+    """
```

Comment:
And thank you for all the great comments/docstrings - it makes it much easier to review!